tools/rtrclient.c

```diff
@@ -106,7 +106,7 @@ static void update_spki(struct spki_table* s __attribute__((unused)), const stru
 
 int main(int argc, char** argv)
 {
-    enum mode_t { TCP, SSH } mode;
+    enum mode_t { NOP, TCP, SSH } mode = NOP;
```

Comment:
Wrt readability it's better to terminate early, I would say. Concur with @fho suggestion, and remove lines 173-176.